django/applications/catmaid/models.py

```diff
@@ -850,13 +847,15 @@ def get_subclass_links_qs():
                 num_linked_ci = self.get_num_class_instances(ci, link.class_a_id)
                 if num_linked_ci != self.value:
                     return True
+            return False
```

Comment:
The logic here was incomplete and it was possible to fall out of the function without a return

Reply:
:+1: